### DISCUSSION TITLE
编译到kernel提示这些怎么解决

### DISCUSSION THREAD

**Discussion by lxdklp** [CmST0us/tspi-linux-sdk-archived#13] [Q&A] [answered]
# 求助，系统为Ubuntu22.04.3lts
==========================================
          Start building kernel
==========================================
+ make -C kernel/ -j17 CROSS_COMPILE=/home/lxdklp/tspi-linux-sdk/prebuilts/gcc/linux-x86/aarch64/gcc-arm-10.3-2021.07-x86_64-aarch64-none-linux-gnu/bin/aarch64-none-linux-gnu- ARCH=arm64 rockchip_linux_defconfig rockchip_linux_docker.config
make: Entering directory '/home/lxdklp/tspi-linux-sdk/kernel'
/home/lxdklp/tspi-linux-sdk/kernel/scripts/gcc-wrapper.py /home/lxdklp/tspi-linux-sdk/prebuilts/gcc/linux-x86/aarch64/gcc-arm-10.3-2021.07-x86_64-aarch64-none-linux-gnu/bin/aarch64-none-linux-gnu-gcc: unknown assembler invoked
scripts/Kconfig.include:47: Sorry, this assembler is not supported.
make[2]: *** [scripts/kconfig/Makefile:88: rockchip_linux_defconfig] Error 1
make[1]: *** [Makefile:639: rockchip_linux_defconfig] Error 2
make: *** [Makefile:357: __build_one_by_one] Error 2
make: Leaving directory '/home/lxdklp/tspi-linux-sdk/kernel'
ERROR: Running /home/lxdklp/tspi-linux-sdk/device/rockchip/common/scripts/mk-kernel.sh - run_command failed!
ERROR: exit code 2 from line 36:
    $@
ERROR: call stack:
    build-helper: run_command(36)
    mk-kernel.sh: do_build(46)
    mk-kernel.sh: build_hook(130)
    mk-kernel.sh: main(202)
ERROR: Running /home/lxdklp/tspi-linux-sdk/device/rockchip/common/build-hooks/99-all.sh - build_all failed!
ERROR: exit code 2 from line 26:
    "$SCRIPTS_DIR/mk-kernel.sh"
ERROR: call stack:
    99-all.sh: build_all(26)
    99-all.sh: build_allsave(107)
    99-all.sh: build_hook(132)
    build-helper: try_func(63)
    build-helper: try_hook(95)
    build-helper: source(158)
    99-all.sh: main(142)
ERROR: Running /home/lxdklp/tspi-linux-sdk/device/rockchip/common/build-hooks/99-all.sh - try_func build_hook allsave failed!
ERROR: exit code 2 from line 67:
    build_hook
ERROR: call stack:
    build-helper: try_func(67)
    build-helper: try_hook(95)
    build-helper: source(158)
    99-all.sh: main(142)
ERROR: Running ./build.sh - run_hooks build allsave failed!
ERROR: exit code 2 from line 152:
    /home/lxdklp/tspi-linux-sdk/device/rockchip/common/build-hooks/99-all.sh build allsave
ERROR: call stack:
    build.sh: run_hooks(152)
    build.sh: run_build_hooks(171)
    build.sh: main(437)
    build.sh: main(452)
ERROR: Running ./build.sh - run_build_hooks build allsave failed!
ERROR: exit code 2 from line 174:
    build
ERROR: call stack:
    build.sh: run_build_hooks(174)
    build.sh: main(437)
    build.sh: main(452)

**Comment by CmST0us** [comment]
请问安装了git-lfs了吗

**Comment by CmST0us** [comment]
我刚刚看到现在这个仓库的git-lfs流量满了, 估计git-lfs 的内容会拉不下来. 建议你下载源码包, 来编译

**Reply by lxdklp** [reply]
我已经下载了百度网盘里面的源码包，提示一样

**Comment by lxdklp** [comment, marked as answer]
又执行了一边git lfs pull 可以了

**Reply by CmST0us** [reply]
因为昨天仓库的带宽满了..git-lfs 被停掉了
